general-information.md

```diff
@@ -56,3 +84,111 @@ Users of beta features are strongly encouraged to share their experiences, learn
+| `human`           | Pedal or foot propulsion                               |
+| `electric_assist` | Provides power only alongside human propulsion         |
+| `electric`        | Contains throttle mode with a battery-powered motor    |
+| `combustion`      | Contains throttle mode with a gas engine-powered motor |
+
+A vehicle may have one or more values from the `propulsion`, depending on the number of modes of operation. For example, a scooter that can be powered by foot or by electric motor would have the `propulsion` represented by the array `['human', 'electric']`. A bicycle with pedal-assist would have the `propulsion` represented by the array `['human', 'electric_assist']` if it can also be operated as a traditional bicycle.
+
+## Vehicle States
+
+This table describes the list of vehicle conditions that may be used by regulators to assess the disposition of individual vehicles and fleets of vehicles.  Some of these states describe vehicles in the Public Right-of-Way (PROW), and others represent vehicles that are not.  One state (`unknown`) implies that PROW status is unknown.
+
+In a multi-jurisdiction environment, the status of a vehicle is per-jurisdiction.  For example, a vehicle may be in the `trip` status for a county that contains five cities, and also in the `trip` status for one of those cities, but `elsewhere` for the other four cities.  In such a condition, generally a Provider would send the device data to the over-arching jurisdiction (the county) and the vehicle state with respect to each city would be determined by the Agency managing the jurisdictions.
+
+| `vehicle_state` | In PROW? | Description |
+| --- | --- |
+| `removed`     | no | Examples include: at the Provider's warehouse, in a Provider's truck, or destroyed and in a landfill. |
+| `available`   | yes | Available for rental via the Provider's app. In PROW. |
+| `unavailable` | yes | Not available for rent.  Examples include: vehicle has low battery, or currently outside legal operating hours. |
+| `reserved`    | yes | Reserved via Provider's app, waiting to be picked up by a rider. |
+| `trip`        | yes | In posession of renter.  May or may not be in motion. |
+| `elsewhere`   | no | Outside of regulator's jurisdiction, and thus not subject to cap-counts or other regulations. Example: a vehicle that started a trip in L.A. has transitioned to Santa Monica.  |
+| `unknown`     | unknown | Provider has lost contact with the vehicle and its disposition is unknown.  Examples include: taken into a private residence, thrown in river. |
+
+## Vehicle Events
+
+This is the list of `vehicle_state` and `event_type` pairings that constitute the valid transitions of the vehicle state machine.
+
+Note that to handle out-of-order events, the validity of the prior-state is not enforced at the time of ingest via Provider or Agency.  Events received out-of-order may result in transient incorrect vehicle states.
```

Comment:
I know this was simply moved from the agency spec but this phrase "prior-state is not enforced" reads like a description of an implementation rather than a specifying behavior that should be implemented. Something like "prior-state shall not be enforced" makes this clearer to me.

Reply:
Changed

Reply:
Agree, fixed.